lessons/module-2/closures.md

```diff
@@ -8,12 +8,19 @@ module: 2
 
 * Describe and speak to lexical scoping in JavaScript
 * Understand and implement closures
+* Identify scenarios where closures can be useful in code, and use closures to solve specific programming challenges
```

Comment:
Love this - seeing scenarios where they are useful is going to help clear up a lot for students